### PR TITLE
#19660 Rename graphql-page-api types. Fix caching issues

### PR DESCRIPTION
**Pull request by dsilvam**

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/graphql/CustomFieldType.java [hunk 1]
```diff
@@ -43,6 +43,24 @@
...
+
+    public static final String DOT_KEY_VALUE = "DotKeyValue";
+
+    public static final String DOT_LANGUAGE = "DotLanguage";
+
+    public static final String DOT_USER = "DotUser";
+
+    public static final String DOT_FILEASSET = "DotFileasset";
+
+    public static final String DOT_SITE = "DotSite";
```
Comment (freddyucv) [inline dotCMS/src/main/java/com/dotcms/graphql/CustomFieldType.java:62]: should it be a enum?

File: dotCMS/src/main/java/com/dotcms/graphql/business/GraphQLTypesProvider.java [hunk 2]
```diff
@@ -18,4 +18,9 @@
      * @throws DotDataException in case of invalid data
      */
     Collection<? extends GraphQLType> getTypes() throws DotDataException;
+
+    /**
+     * Flushes any
+     */
+//    void invalidateTypes();
```
Comment (freddyucv) [inline dotCMS/src/main/java/com/dotcms/graphql/business/GraphQLTypesProvider.java:25]: why is this code comment?

File: dotCMS/src/main/java/com/dotcms/graphql/business/PageAPIGraphQLTypesProvider.java [hunk 3]
```diff
@@ -63,10 +67,30 @@
 
     INSTANCE;
 
-    Map<String, GraphQLOutputType> typeMap = new HashMap<>();
+    public static final String DOT_PAGE = "DotPage";
```
Comment (victoralfaro-dotcms) [inline dotCMS/src/main/java/com/dotcms/graphql/business/PageAPIGraphQLTypesProvider.java:70]: Maybe an enum for these values?

File: dotCMS/src/main/java/com/dotcms/graphql/CustomFieldType.java (hunk 1 above)
Comment (dsilvam) [reply dotCMS/src/main/java/com/dotcms/graphql/CustomFieldType.java]: Done

File: dotCMS/src/main/java/com/dotcms/graphql/business/GraphQLTypesProvider.java (hunk 2 above)
Comment (dsilvam) [reply dotCMS/src/main/java/com/dotcms/graphql/business/GraphQLTypesProvider.java]: Done

**Review by freddyucv — APPROVED** [review]

**Review by victoralfaro-dotcms — APPROVED** [review]
LGTM

**Pull request merged**